.travis.yml

```diff
@@ -12,24 +12,64 @@ matrix:
       env: TOXENV=py37-django22
     - python: 3.8
       env: TOXENV=py38-django22
-    - python: 3.9
+    - python: 3.9-dev
       env: TOXENV=py39-django22
     - python: 3.6
       env: TOXENV=py36-django30
     - python: 3.7
       env: TOXENV=py37-django30
     - python: 3.8
       env: TOXENV=py38-django30
-    - python: 3.9
+    - python: 3.9-dev
```

Comment:
i think we can use python 3.9 now instead of dev

Reply:
Hi @auvipy , I had replaced the version of python as 3.9 as well as removed test for python 3.5. Hope it is fine. Please have a look.
Thanks!